app/frontend/src/store/modules/form.js

```diff
@@ -120,6 +122,7 @@ export default {
     updateError: state => state.updateError,
 
     // Form objects
+    operationType: state => state.ipcPlan.operationType,
```

Comment:
This is part of the IPCPlan object so should use the IPC plan getter and setter rather than its own.

Reply:
oh yes.. i should have thought of that!